app/portainer/components/datatables/datatable.css

```diff
@@ -278,3 +278,12 @@
   border: none;
   background: none;
 }
+
+.actions-menu-btn {
```

Comment:
is this rule required? https://github.com/portainer/portainer/blob/7c36a275a84371dca73dcf78442f834b68598d4f/app/portainer/components/datatables/components/ActionsMenu.module.css#L22

Reply:
removed it